src/foundry/foundry.js/applications/sidebarTabs/settings.d.ts

```diff
@@ -35,21 +44,4 @@ declare namespace Settings {
     coreUpdate: string | false;
```

Comment:
- `user` should be the configured document class.
- `isDemo` is missing
- `systemUpdate` is missing